app_feup/lib/view/Widgets/generic_card.dart

```diff
@@ -94,7 +94,16 @@ class GenericCardState extends State<GenericCard> {
                             padding: EdgeInsets.symmetric(horizontal: 15),
                             margin: EdgeInsets.only(top: 15, bottom: 10),
                           )),
-                          this.getDeleteIcon(context)
+                          Container(
+                            child: this.getMoveIcon(context),
+                            alignment: Alignment.center,
+                            margin: EdgeInsets.only(top: 15, bottom: 10),
```

Comment:
The icon is vertically center aligned. But because the title only uses upper case in the first letter the icon looks visually disheveled. 
<div>
<img width="300" alt="version default" src="https://user-images.githubusercontent.com/53405284/154857134-0b4199b6-6fca-439f-9abf-13b4e1c342c8.png">
<img width="300" alt="Captura de ecrã 2022-02-20, às 17 53 35" src="https://user-images.githubusercontent.com/53405284/154857136-6b0d9f92-721d-4f3c-bfdf-7ad17e4cdd08.png">
</div>

If we apply this new margin can be like:

<div>
<img width="300" alt="version 1" src="https://user-images.githubusercontent.com/53405284/154857203-7b8b4818-f370-4b34-8dd0-e70a67551e95.png">
<img width="300" alt="Captura de ecrã 2022-02-20, às 17 54 09" src="https://user-images.githubusercontent.com/53405284/154857206-26f14d60-f12c-43d9-a891-cbaac5d0f7cf.png">
</div>

@ines-08 What do you think? Should we apply a new margin or keep the other one?

```suggestion
                            margin: EdgeInsets.only(top: 8),
```

Reply:
I like the new one you suggested better